.github/workflows/wintest.yml

```diff
@@ -18,14 +16,13 @@ jobs:
     strategy:
       matrix:
         # Cannot test OTP 21 due to https://bugs.erlang.org/browse/ERL-644 (see also https://github.com/elixir-lang/elixir/issues/7774)
-        otp_version: [19, 20, 22, 23]
+        otp_version: [22, 23, 24]
```

Comment:
So, `erlef/setup-beam` is not 21- -compatible, but neither is `rebar3` (latest) as of this moment, so I guess this change is Ok.